prefs.js

```diff
@@ -111,10 +112,37 @@ function buildPrefsWidget() {
     Gio.SettingsBindFlags.DEFAULT
   );
   prefsWidget.attach(feedbackSoundsSwitch, 1, 4, 1, 1);
+  feedbackSoundsSwitch.set_sensitive(isPlayingSoundSupported);
+
+  if (!isPlayingSoundSupported) {
+    const playingSoundNotSupportedLabel = new Gtk.Label({
+      halign: Gtk.Align.START,
+    });
+    playingSoundNotSupportedLabel.set_markup("<span foreground='red'>WARNING. Playing sound is not supported on this system. Is GStreamer package installed?</span>");
+    if (GTK_VERSION == 3) {
+      playingSoundNotSupportedLabel.set_line_wrap(true);
+    } else {
+      playingSoundNotSupportedLabel.set_wrap(true);
+    }
+    prefsWidget.attach(playingSoundNotSupportedLabel, 0, 5, 1, 1);
+  }
 
   if (GTK_VERSION == 3) {
     prefsWidget.show_all();
   }
 
   return prefsWidget;
 }
+
+function is_playing_sound_supported() {
+  try {
+    imports.gi.Gst;
+    imports.gi.GstAudio;
+    return true;
+  } catch(e) {
+    log(`${Extension.metadata.uuid}: Playing sound is not supported on this system. Is GStremer package installed?`);
```

Comment:
typo: GStremer -> GStreamer

Reply:
Hehe, I fixed it already in the extensions.js :).